tests/Css/StyleTest.php

```diff
@@ -51,7 +52,43 @@ public static function lengthInPtProvider(): array
             ["calc((50% + 10) 1pt)", 100, 0.0],            // invalid - missing op
             ["calc(50% -1pt)", 100, 0.0],                  // invalid - missing op
             ["calc((50% + 10) + 2pt))", 100, 0.0],         // invalid - extra bracket
-            ["calc(100pt / 0)", null, 0.0]                 // invalid - division by zero
+            ["calc(100pt / 0)", null, 0.0],                // invalid - division by zero
+
+            // Comparison Functions
+            ["min(-20, 5 * 2, 8)", null, -20.0],           // min function
+            ["max(-20, 5 * 2, 8)", null, 10.0],            // max function
+            ["clamp(10, 15 - 7, 20)", null, 10.0],         // clamp min function
+            ["clamp(10, 15 + 7, 20)", null, 20.0],         // clamp max function
+            ["clamp(10, 7 * 2, 20)", null, 14.0],          // clamp val function
+
+            // Stepped Value Functions
+            ["round(12.31, 1)", null, 12.3],               // complex
+            ["round(down, 12.31, 1)", null, 0.0],          // Yet not supported!!!
+            ["mod(7, 2)", null, 1.0],                      // mod function
+            ["rem(21, 6)", null, 3.0],                     // rem function
+
+            // Trigonometric Functions
+            // Can't test because different calc precision
+            // "sin","cos","tan","asin","acos","atan","atan2",
```

Comment:
How about testing edge cases that are certain to have an integer result, e.g. `sin(0) == 0`, `cos(0) == 1` etc? For other cases, it should work to allow the result to match the expected value with an error margin: You could allow `testLengthInPt` to round the result with an optional fourth parameter, and use that for the trigonometric functions.